### PR TITLE
18254 - AUTH - Backend - Environment variables cleanup

### PR DESCRIPTION
**Pull request by sameer0422**
*Issue #:*
https://github.com/bcgov/entity/issues/18254

*Description of changes:* AUTH - Backend - Environment variables cleanup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ ENV VCS_REF=${VCS_REF}
 ENV BUILD_DATE=${BUILD_DATE}
 ENV PORT=8080
 
+CMD gunicorn --bind 0.0.0.0:${PORT} --config /code/gunicorn_config.py wsgi:app
```
Comment (seeker25) [inline auth-api/Dockerfile:10]: Hmm this doesn't look right

Comment (avni-work) [reply auth-api/Dockerfile:10]: yeah we don't want this changed. this portion of the ticket is already done @sameer0422

**Review by avni-work — CHANGES_REQUESTED** [review]
we are only focusing on the following changes:
  ```
  # POSTGRESQL
    if DB_UNIX_SOCKET := os.getenv("DATABASE_UNIX_SOCKET", None):
        SQLALCHEMY_DATABASE_URI = f"postgresql+psycopg2://{DB_USER}:{DB_PASSWORD}@/{DB_NAME}?host={DB_UNIX_SOCKET}"
    else:
        SQLALCHEMY_DATABASE_URI = f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
```

File: auth-api/Dockerfile [hunk 2]
```diff
@@ -72,4 +72,4 @@ USER web
 FROM development_build AS production_build
 COPY --chown=web:web . /code
 
-CMD gunicorn --bind 0.0.0.0:${PORT} --config /code/gunicorn_config.py wsgi:app
+CMD gunicorn --bind 0.0.0.0:${PORT} --config /code/gunicorn_config.py wsgi:app
```
Comment (thorwolpert) [inline auth-api/Dockerfile]: Can you just remove the Dockerfile from your PR?

**Review by thorwolpert — COMMENTED** [review]
Looks good, so just remove the Dockerfile from the PR and it's good to go.

File: auth-api/Dockerfile (hunk 2 above)
Comment (avni-work) [reply auth-api/Dockerfile]: yes just helped him revert his changes

Comment (sameer0422) [reply auth-api/Dockerfile]: Fixed

**Review by avni-work — APPROVED** [review]

**Comment by seeker25** [comment]
If you remove the docker file from the PR, Openshift wont deploy anymore

**Comment by seeker25** [comment]
@sameer0422 add this in 
https://github.com/bcgov/sbc-pay/blob/71ccff03766adc0e8a08579e5207d3bff2221504/.github/workflows/pay-api-ci.yml#L96

**Review by avni-work — APPROVED** [review]

**Pull request merged**